docs/guide/introduction.rst

```diff
@@ -0,0 +1,300 @@
+
+This is your bot instance. You can name the variable whatever you like but it is customary to name :class:`~ext.commands.Bot` instances ``bot``.
+You will need to specify a ``command_prefix`` here, we use ``!``, but you can use any string, or a list of strings for multiple prefixes.
+We also add the :attr:`Intents.message_content` intent, so that our bot can read regular text messages. You can read more in the `Intents guide <_guide_intents>`_.
+
+.. note::
+    This use of :class:`~ext.commands.Bot` is using the library provided implementation of Bot. For extending usage you may wish to consider subclassing it.
+
+
+.. code-block:: python
+
+    @bot.listen()
+    async def on_ready():
+        print(f'Ready! I am {bot.user} and my ID is {bot.user.id}')
+
+This is the :func:`on_ready` event. It is called when the bot has finished loading and everything necessary is cached.
+We use the :meth:`@bot.listen() <ext.commands.Bot.listen>` decorator as to not override the main event.
+For a list of available events, see :ref:`discord-api-events`.
+
+Since you will likely want to override other events, such as the one that exists for any new message the bot can read, you should consider this warning `here <ext_commands_on_message>`.
+
+.. warning::
+
+    :func:`on_ready` can and will be called multiple times throughout your bot's uptime. You should avoid doing any kind of state-management here, such as connecting and loading your database.
+
+    Consider using :meth:`~ext.commands.Bot.setup_hook` for this purpose instead.
+
+Moving on further, let's now create a basic command that can take further input from the person sending the message:-
+
+.. code-block:: python
+
+    @bot.command()
+    async def apples(ctx: commands.Context, amount: int) -> None:
+        await ctx.send(f"Hello, I would like {amount} apples please!")
+
+Here's where our commands will be defined. We use the :meth:`@bot.command() <ext.commands.Bot.command>` decorator to flag this function as a command.
+This creates a command ``!apples`` that we can type into a channel, and the bot will respond with the given amount of applies in a predefined sentence.. A few key features:
+
+- :class:`~ext.commands.Context` refers to the command invocation context - this includes the :attr:`~ext.commands.Context.channel`, command :attr:`~ext.commands.Context.author`, :attr:`~ext.commands.Context.message` and more.
+- Using ``int`` as a parameter type annotation here will instruct the library to attempt the given argument to an :class:`int` type. This means you'll have an integer in the command body and not a string.
+
+For more information on the commands extension and other converters, please reference :ref:`ext_commands_commands`.
+
+.. code-block:: python
+
+    TOKEN = "your bot token here"
+
+This is the final step, you put your bot token here, save and run the file and the bot will start up.
```

Comment:
nowhere in this do I see Bot.run explicitly being called. you'll need to add that.